### ISSUE TITLE
Data access

### ISSUE THREAD

**Issue by Jiaxin-Wen** [lingo-mit/deductive-closure#6]
Hey!  Could you directly upload the generated data? It'd be super helpful!

**Comment by feyzaakyurek** [comment]
Hi Jiaxin,

Is there any particular dataset/experiment you are interested in? Otherwise, I'll try to organize and upload them soon. There are quite many files across different experiments and seeds.

**Comment by Jiaxin-Wen** [comment]
yeah I'm particularly interested in the generated consistency data on CREAK

**Comment by feyzaakyurek** [comment]
Hi Jiaxin,

Just finished uploading the sampled data [here](https://drive.google.com/drive/folders/1Qac61vX36PZwn7rgZPNAwV8T2cDTYqei?usp=drive_link) under generated_data. Each respective folder has a csv file that is used for training called creak_train_finetuning.csv--truth_value field is the label--that is used for fine-tuning. The files all_graphs.txt contain graphs and all_nodes.csv lists claims in separate lines.

Hope this is still useful!

**Comment by Jiaxin-Wen** [comment]
Thank you so much! Happy holidays!

**Comment by Jiaxin-Wen** [comment]
Hi, one qq: for the unsupervised results in the first row of Table 1, are you fine-tuning models or just conducting few-shot prompting?

**Comment by Jiaxin-Wen** [comment]
for more context: 
I'm conducting some similar experiments
- convert question-answer pairs in TruthfulQA into claims, and ask LMs to do binary classification, i.e., judge whether the answer is correct.
- I also use LMs to generate new question-answer pairs. However, when using these generated question-answer pairs as in-context examples, the binary classification performance decreases

I'd like to know if you face similar issues when using the generated claims as in-context examples

**Comment by feyzaakyurek** [comment]
For all DCT experiments we fine-tune the model on the generated data. The first row in Table 1 is the prompting baseline which is few-shot prompting. So, we didn't prompt the model using the generated data.

**Comment by Jiaxin-Wen** [comment]
Thanks for clarifying! Would you have time to run some prompting experiments using the generated data?
